src/client/app/game.service.ts

```diff
@@ -381,6 +381,7 @@ export class GameService {
     });
 
     this.socketService.register(ServerEventName.ChatPlayerListSync, (players) => {
+      // @ts-ignore
```

Comment:
what is this?

Reply:
It suppresses the tslint error on next line ![1](https://i.imgur.com/oDTBW1J.png).  
You mentioned on discord to ignore it https://discordapp.com/channels/221407771785035777/411559762971590667/680292426459512881

Reply:
Sorry! I meant to totally ignore the error. I have no idea why you're seeing it.

Reply:
Okay sure thing